CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ### Bugfixes
 
-* Lorem ipsum.
+* Fixes bug in encryption that could cause deadlocks/hangs and possibly
+  other bugs too. Fixes https://github.com/realm/realm-core/pull/2650
```

Comment:
@finnschiermer There is nothing wrong with it, but as @nirinchev has observed a similar behaviour and *not* doing a file format upgrade, I imagine that we need a more general test.